bin/cli.js

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env node
+/* eslint-disable no-console, import/no-commonjs */
+const argv = require('minimist')(process.argv.slice(2), {
+  string: ['output'],
+  boolean: ['help'],
+});
+const fs = require('fs');
+const storyJsonToAmp = require('../dist').default;
+
+function usage() {
+  console.log('USAGE: story-json-to-amp [options] <story.json>');
+  console.log('');
+  console.log('OPTIONS:');
+  console.log('  --help                   Show this message');
+  console.log('  --output                 Write AMP HTML to file');
```

Comment:
i think `File to write AMP HTML to` is clearer

Reply:
I think "leave blank" can be confusing as this param is optional
"log" is pretty vague, especially for users that aren't familiar with javascript terms, I'd use something like "print to standard output"

Overall usage is meant to be a very short description, longer explanation can be added the to readme